ui/cypress.json

```diff
@@ -1,4 +1,5 @@
 {
   "integrationFolder": "cypress/e2e",
-  "defaultCommandTimeout": 30000
+  "defaultCommandTimeout": 30000,
+  "video": false
```

Comment:
Why this change?

Reply:
The last paragraph in my (very long) PR description goes into it in more detail, but basically running the e2e in a VM made the tests take longer and also made the video encoding actually work...which made them take even longer. At this point we've not had videos working on the e2e tests for so long that I didn't think it was worth turning them on and extending the duration of the tests. When we move to `monitor-ci` there's a bunch of optimization for running the tests in parallel and videos will work great.